installer/conf/omsagent.d/wlm_baseOS.conf

```diff
@@ -55,3 +55,62 @@
   tag oms.wlm.monitor
 </source>
 
+<source>
+  type wlm_oms_omi
+  object_name "Memory"
+  instance_regex ".*"
+  counter_name_regex ".*"
+  interval 30s
+  omi_mapping_path %CONF_DIR_WS%/wlm_monitor_mapping.json
+  tag oms.wlm.monitor
+  wlm_ip "infrastructureInsights"
+  wlm_data_type "WLM_PERF_DATA_BLOB"
+</source>
+
+<source>
+  type wlm_oms_omi
+  object_name "Logical Disk"
+  instance_regex ".*"
+  counter_name_regex ".*"
+  interval 30s
+  omi_mapping_path %CONF_DIR_WS%/wlm_monitor_mapping.json
+  tag oms.wlm.monitor
+  wlm_ip "infrastructureInsights"
+  wlm_data_type "WLM_PERF_DATA_BLOB"
+</source>
+
+<source>
+  type wlm_oms_omi
+  object_name "Physical Disk"
+  instance_regex ".*"
+  counter_name_regex ".*"
+  interval 30s
+  omi_mapping_path %CONF_DIR_WS%/wlm_monitor_mapping.json
+  tag oms.wlm.monitor
+  wlm_ip "infrastructureInsights"
```

Comment:
<nit> i in the IP name should be in caps.

Reply:
This is one of the asks from the service to have it as a small "i" in the packet being sent from the Agent.